modules/metering-writing-reports.adoc

```diff
@@ -36,11 +36,11 @@ spec:
   query: namespace-cpu-request <1>
   runImmediately: true <3>
 ----
-<1> The `query` specifies `ReportQuery` used to generate the report. Change this based on what you want to report on. For a list of options, run `oc get reportqueries | grep -v raw`.
-<2> The `metadata.name` of the report should be something descriptive about what the report does. A good name is the query, and the schedule or period you used.
+<1> The `query` specifies ReportQuery used to generate the Report. Change this based on what you want to Report on. For a list of options, run `oc get reportqueries | grep -v raw`.
+<2> The `metadata.name` of the Report should be something descriptive about what the Report does. A good name is the query, and the schedule or period you used.
```

Comment:
Rewrite to not use "should."
> ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.